packages/react-components/react-color-picker-preview/library/src/components/ColorArea/ColorArea.test.tsx

```diff
@@ -7,12 +7,22 @@ describe('ColorArea', () => {
   isConformant({
     Component: ColorArea,
     displayName: 'ColorArea',
+    disabledTests: ['component-handles-ref', 'component-has-root-ref'],
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
   it('renders a default state', () => {
-    const result = render(<ColorArea>Default ColorArea</ColorArea>);
-    expect(result.container).toMatchSnapshot();
+    const result = render(<ColorArea color="red" />);
+    expect(result.container).toMatchInlineSnapshot(`
+      <div>
```

Comment:
The snapshot doesn't match to what is described in the component's spec:

```HTML
<div role="group" class="fui-ColorArea" aria-label="Gradient area">
  <input id="sliderX" type="range" class="fui-ColorArea__inputX" value="96" aria-label="saturation">
  <input id="sliderY" type="range" class="fui-ColorArea__inputY" value="0" aria-label="brightness">
  <div class="fui-ColorArea__thumb"></div>
</div>
```

Reply:
I'll return it when I add keyboard events. Those slots are needed for keyboard navigation

Reply:
However, it seems we're missing other elements besides input slots, such as roles and classes. Is this intentional?